src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,10 @@
-const booleanToWord = boolean => {};
 
+const booleanToWord = boolean => {
+    if(boolean === true) {
```

Comment:
Great stuff again. I tend to use a ternary for returns where there are only two simple options eg `boolean ? "Yes" : "No"; `